src/main/scala/gitbucket/core/service/PaginationHelper.scala

```diff
@@ -0,0 +1,15 @@
+package gitbucket.core.service
+
+import scala.util.Try
+
+object PaginationHelper {
+
+  def page(page: Option[String]) = {
+
+    page
+      .flatMap(pageStr => Try(pageStr.toInt).toOption)
+      .map(Math.max(1, _)) // remove negative pages
+      .getOrElse(1)
+  }
```

Comment:
Actually, `Int` parameter handling is not consistent in GitBucket codebase. I would like to introduce more general way which can be used in entire codebase in the future.